packages/@cdk-cosmos/core/src/helpers/remote.ts

```diff
@@ -180,11 +180,7 @@ export class RemoteCluster {
   static import(scope: Construct, id: string, exportName: string, vpc: IVpc): ICluster {
     const clusterName = Fn.importValue(`${exportName}Name`);
     const securityGroups = [
-      SecurityGroup.fromSecurityGroupId(
-        scope,
-        `${exportName}SecurityGroup`,
-        Fn.importValue(`${exportName}SecurityGroup`)
-      ),
+      SecurityGroup.fromSecurityGroupId(scope, `${id}SecurityGroup`, Fn.importValue(`${exportName}SecurityGroup`)),
```

Comment:
Will this cause name change of SecurityGroup...? I suppose not as snapshot hasn't changed.

Reply:
only if your creating constructs under the portal